modules/auth/ldap/ldap.go

```diff
@@ -149,18 +158,39 @@ func bindUser(l *ldap.Conn, userDN, passwd string) error {
 	return err
 }
 
+func checkAdmin(l *ldap.Conn, ls *Source, userDN string) bool {
+	if len(ls.AdminFilter) > 0 {
```

Comment:
Ok, with latest commit it will not toch IsAdmin flag for user if no `AdminFilter` attribute is set in LDAP configuration